lib/portlayer/storage/vsphere/image.go

```diff
@@ -641,3 +661,61 @@ func imagesInUse(op trace.Operation, ID string) error {
 
 	return nil
 }
+
+// populate the scratch with minimum OS structure defined in FileForMinOS and DirForMinOS
+func populateMinOS(op trace.Operation, vmdisk *disk.VirtualDisk) error {
+	// tmp dir to mount the disk
+	dir, err := ioutil.TempDir("", "mnt-"+portlayer.Scratch.ID)
+	if err != nil {
+		op.Errorf("Failed to create tempDir: %s", err)
+	}
+	defer os.RemoveAll(dir)
+
+	if err = vmdisk.Mount(dir, nil); err != nil {
+		op.Errorf("Failed to mount device %s to dir %s", vmdisk.DevicePath, dir)
+	}
+
+	for dname, dmode := range DirForMinOS {
+		dirPath := fmt.Sprintf("%s%s", dir, dname)
```

Comment:
`path.Join(dir, dname)` - adds system appropriate path separator and makes it obvious it's path manipulation rather than just string manipulation